src/test/java/edu/ucsb/cs156/rec/controllers/RequestTypesControllerTests.java

```diff
@@ -47,215 +47,51 @@ public class RequestTypesControllerTests extends ControllerTestCase {
         @MockBean
         UserRepository userRepository;
 
-        // Authorization tests for /api/phones/admin/all
-
-
-        @Test
-        public void logged_out_users_cannot_get_all() throws Exception {
-                mockMvc.perform(get("/api/requesttypes/all"))
-                                .andExpect(status().is(403)); // logged out users can't get all
-        }
```

Comment:
I think you need to integrate both the old and new code together. It appears that you are deleting code from one of your teammates (or an earlier PR of yours?) and replacing it ... but that's going to undo some of the work for which the team already earned points. 

So that's not going to work.